README.md

```diff
@@ -1,7 +1,222 @@
+
+
+<!-- PROJECT SHIELDS -->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+
+
+<!-- PROJECT LOGO photo -->
+<br />
+<p align="center">
+  <a href="https://github.com/Shelf-Share-BEE-Mod3">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">Shelf Share</h3>
+
+  <p align="center">
+    ShelfShare is a platform for sharing books via mail. You can borrow and lend books with your friends!
+    <br />
+    <a href="https://github.com/Shelf-Share-BEE-Mod3"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="Add our video link here">View Demo</a>
+    ·
+    <a href="https://github.com/Shelf-Share-BEE-Mod3/shelf-share-rails/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/Shelf-Share-BEE-Mod3/shelf-share-rails/issues">Request Feature</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Usage](#usage)
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+* [License](#license)
+* [Contact](#contact)
+* [Acknowledgements](#acknowledgements)
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+[![about-screenshot][about-screenshot]](https://example.com)
+
+ShelfShare wants to help you save money by allowing you to borrow and lend out books you own to your friends. We put those dusty books in your collection to use!
+
+Here's why:
+* Most of us are guilty of buying that book we "have to have" just to read it once and stash it on a shelf somewhere.
+* Save money! Shelf share is a free service. Once you find a book you are interested in on a friend's shelf, request to borrow the book, pay for shipping, and you are ready to go!
+* Convenience. ShelfShare is simplistic in design, no walking around the library trying to find a book. Simply look through the books that your friends own and request a book! Easy!  :smile:
+
+
+A list of commonly used resources and references that our team found helpful in the creation of this project are listed in acknowledgements.
+
+### Built With
+
+* [Sinatra](https://github.com/sinatra/sinatra)
+* [Rails](https://github.com/rails/rails)
+* [Ruby](https://github.com/ruby/ruby)
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To use ShelfShare locally, you can fork or clone [these](https://github.com/Shelf-Share-BEE-Mod3) two repos.
+The **ShelfShare Web App** is the application itself, while **BookBuddy** is the micro-service we will be using to find the book information we need to add books to our shelf!
```

Comment:
This may be a good spot to link to the BookBuddy repo

Reply:
Book Buddy is located in that link.